cmd/revoke_listactivation_test.go

```diff
@@ -64,5 +64,112 @@ func TestRevokeListActivation(t *testing.T) {
 	require.True(t, strings.Contains(stdout, pub))
 	require.False(t, strings.Contains(stdout, time.Unix(1001, 0).Format(time.RFC1123)))
 	require.True(t, strings.Contains(stdout, time.Unix(2001, 0).Format(time.RFC1123)))
+}
+
+func TestRevokeListActivationNoAccount(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	_, _, err := ExecuteInteractiveCmd(createRevokeListActivationCmd(), []interface{}{})
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "no accounts defined")
+}
+
+func TestRevokeListActivationNoAccountInteractive(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	_, _, err := ExecuteCmd(createRevokeListActivationCmd())
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "an account is required")
+}
+
+func TestRevokeListActivationNoExport(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	ts.AddAccount(t, "A")
+	_, _, err := ExecuteCmd(createRevokeListActivationCmd(), "--service")
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "doesn't have exports")
+}
 
+func TestRevokeListActivationNoServiceExport(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	ts.AddAccount(t, "A")
+	ts.AddExport(t, "A", jwt.Stream, "s", false)
+	_, _, err := ExecuteCmd(createRevokeListActivationCmd(), "--service")
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "doesn't have service exports")
+}
+
+func TestRevokeListActivationNoStreamExport(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	ts.AddAccount(t, "A")
+	ts.AddExport(t, "A", jwt.Service, "s", false)
+	_, _, err := ExecuteCmd(createRevokeListActivationCmd())
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "doesn't have stream exports")
+}
+
+func TestRevokeListActivationDefaultExport(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+	ts.AddAccount(t, "A")
+	ts.AddExport(t, "A", jwt.Service, "s", false)
+	_, _, err := ExecuteCmd(createRevokeActivationCmd(), "--service", "--target-account", "*")
```

Comment:
shouldn't we also look at the claim and find, * in the revocations?
Also, the same for tests below

Reply:
> check the actual claims in the unit tests?
> When revoking * for activation and user, shouldn't we perform cleanup of older now redundant keys?

That should be done in the JWT library.

Reply:
https://github.com/nats-io/jwt/issues/126